valiu-node-commons/src/valiu_currencies.rs

```diff
@@ -0,0 +1,35 @@
+use core::fmt;
+
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+#[derive(
+    Clone,
+    Copy,
+    Debug,
+    Eq,
+    Ord,
+    PartialEq,
+    PartialOrd,
+    parity_scale_codec::Decode,
+    parity_scale_codec::Encode,
+)]
+pub enum ValiuCurrencies {
```

Comment:
Complementing the other comment, what do you say about in general using better the word asset instead of currency since it's a bit more general and we don't know if our currency will be backed by Dollars, houses or potatoes?

Reply:
Hehehehe, potatoes was funny.

Sure. `Asseet` will be